modules/sovrnBidAdapter.js

```diff
@@ -95,7 +95,22 @@ export const spec = {
       });
     }
     return sovrnBidResponses;
-  }
+  },
+
+  getUserSyncs: function(syncOptions, serverResponses) {
+    if (serverResponses && serverResponses.length !== 0 && syncOptions.iframeEnabled) {
+      let iidArr = serverResponses.filter(rsp => rsp.body && rsp.body.ext && rsp.body.ext.iid)
```

Comment:
not super sure this will work without parenthesis, have you tested with rsp.body and not rsp.body.ext ?

Reply:
Yup, tested this case. Works fine from what I saw.